src/utils/institutions-helpers.ts

```diff
@@ -12,3 +12,11 @@ export const filterInstitutions = (
     return !excludeIds.includes(institution.id);
   });
 };
+
+// Find the most specific unit in hierarchy
+export const getMostSpecificUnit = (values: FormikInstitutionUnit): any => {
```

Comment:
hvis return type er `any` så er det ikke noe vits å sette den på
```suggestion
export const getMostSpecificUnit = (values: FormikInstitutionUnit) => {
```

Reply:
her klager den hvis jeg fjerner:
```
Function implicitly has return type 'any' because it does not have a return type annotation and is referenced directly or indirectly in one of its return expressions.
```

Reply:
ah, sikkert fordi `mostSpecificUnit` i `AffiliationsCell.tsx` ikke har type?

Reply:
det kan være ja!
men burde egentlig spesifisere type her da, så kan jukse til ved å returnere `values as InstitutionUnitBase`